frontend/src/metabase/styled-components/theme/css-variables.ts

```diff
@@ -5,13 +5,32 @@ import type { MetabaseComponentTheme } from "embedding-sdk";
 import { SDK_TO_MAIN_APP_COLORS_MAPPING } from "embedding-sdk/lib/theme/embedding-color-palette";
 import type { MantineTheme } from "metabase/ui";
 
+/** Maps the CSS variable name to the corresponding theme key in the Embedding SDK theme. */
+const CSS_VARIABLES_TO_SDK_THEME_MAP = {
+  "--mb-color-tooltip-text": "tooltip.textColor",
+  "--mb-color-tooltip-background": "tooltip.backgroundColor",
+  "--mb-color-tooltip-background-focused": "tooltip.focusedBackgroundColor",
+  "--mb-color-tooltip-text-secondary": "tooltip.secondaryTextColor",
+  "--mb-color-bg-dashboard": "dashboard.backgroundColor",
+  "--mb-color-bg-dashboard-card": "dashboard.card.backgroundColor",
+  "--mb-color-bg-question": "question.backgroundColor",
+  "--mb-color-text-collection-browser-expand-button":
+    "collectionBrowser.breadcrumbs.expandButton.textColor",
+  "--mb-color-bg-collection-browser-expand-button":
+    "collectionBrowser.breadcrumbs.expandButton.backgroundColor",
+  "--mb-color-text-collection-browser-expand-button-hover":
+    "collectionBrowser.breadcrumbs.expandButton.hoverTextColor",
+  "--mb-color-bg-collection-browser-expand-button-hover":
+    "collectionBrowser.breadcrumbs.expandButton.hoverBackgroundColor",
+} satisfies Record<string, MetabaseComponentThemeKey>;
```

Comment:
The theme key has full autocomplete support thanks to Oisin's use of `FlattenObjectKeys`. I mainly keep the `--mb-` prefix explicit so it's easy to find this `CSS_VARIABLES_TO_SDK_THEME_MAP` map by a full-text search.

![CleanShot 2567-11-26 at 00 50 13@2x](https://github.com/user-attachments/assets/ae67c95f-8169-40cd-aaf6-868590f8cb52)